app/mailers/spotlight/contact_mailer.rb

```diff
@@ -2,7 +2,7 @@
 
 module Spotlight
   # Mailer for reporting problems to the application contact and/or exhibit administrator
-  class ContactMailer < ActionMailer::Base
+  class ContactMailer < ApplicationMailer
```

Comment:
Is this controversial? I'm not sure? I'd offer this is the most straightforward way I could think of for a mailer to inherit application defaults without needing to go and create custom configurations for it.

Reply:
👍 from me; I think this is generated by default these days.